x-pack/plugins/apm/kibana.json

```diff
@@ -3,44 +3,40 @@
   "version": "8.0.0",
   "kibanaVersion": "kibana",
   "requiredPlugins": [
-    "features",
     "apmOss",
     "data",
-    "licensing",
-    "triggersActionsUi",
     "embeddable",
+    "features",
+    "fleet",
     "infra",
+    "licensing",
     "observability",
-    "ruleRegistry"
+    "ruleRegistry",
+    "triggersActionsUi"
   ],
   "optionalPlugins": [
-    "spaces",
-    "cloud",
-    "usageCollection",
-    "taskManager",
     "actions",
     "alerting",
-    "security",
-    "ml",
+    "cloud",
     "home",
     "maps",
-    "fleet"
+    "ml",
+    "security",
+    "spaces",
+    "taskManager",
+    "usageCollection"
   ],
   "server": true,
   "ui": true,
-  "configPath": [
-    "xpack",
-    "apm"
-  ],
-  "extraPublicDirs": [
-    "public/style/variables"
-  ],
+  "configPath": ["xpack", "apm"],
+  "extraPublicDirs": ["public/style/variables"],
```

Comment:
FYI I'm about to merge a PR that removes this line so you could have conflicts.

Reply:
Thanks for the note. Merged upstream and fixed the conflict here.